manifest/dev.baru/HeadCullingMask/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "HeadCullingMask",
+	"id": "net.hantabaru1014.HeadCullingMask",
```

Comment:
Doesn't match author folder of `dev.baru`

Reply:
Can one author create multiple folders?
I've separated the folders for now

Reply:
That works, can consolidate those to one or the other in the future.